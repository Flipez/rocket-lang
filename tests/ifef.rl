i = 0
while (i<2)
  if (i==0)
    puts("if/ef: if block")
  ef (i==1)
    puts("if/ef: ef block")
  end

  i = i+1
end
