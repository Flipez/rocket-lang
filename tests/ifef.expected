"if/ef: if block"
"if/ef: ef block"
nil
